adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -1957,6 +1974,9 @@ public void testAutoFlowRefreshTokenRequestFailedWithOauthError() throws IOExcep
                  AuthenticationConstants.UIResponse.BROWSER_CODE_COMPLETE,
                  getResponseIntent(callback, "resource", "clientid", "redirect", TEST_IDTOKEN_UPN));
          signalCallback.await(CONTEXT_REQUEST_TIME_OUT, TimeUnit.MILLISECONDS);
+
+         assertTrue(telemetry.checkOauthError());
```

Comment:
yes, will do in the next iteration of telemetry. this is solely focused on getting the bits to hands of OneDrive.

---
In reply to: [91187483](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r91187483) [](ancestors = 91187483)